### ISSUE TITLE
`wellKnownKustoEndpoints.json` missing from 4.0.0 source on pypi

### ISSUE THREAD

**Issue by xylar** [Azure/azure-kusto-python#435]
#### Code Sample, a copy-pastable example if possible

```bash
conda create -y -n test python=3.10
conda activate test
wget https://files.pythonhosted.org/packages/50/48/84420f85f584a1e2d13313addf950cfc9b8bdbee36cc05a172792c4430bd/azure-kusto-data-4.0.0.tar.gz
tar xvf azure-kusto-data-4.0.0.tar.gz
cd azure-kusto-data-4.0.0/
mamba activate test
pip install .
python -c "import azure.kusto.data"
```

Error message:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/xylar/mambaforge/envs/test/lib/python3.10/site-packages/azure/kusto/data/__init__.py", line 5, in <module>
    from .client import KustoClient
  File "/home/xylar/mambaforge/envs/test/lib/python3.10/site-packages/azure/kusto/data/client.py", line 13, in <module>
    from .client_base import ExecuteRequestParams, _KustoClientBase
  File "/home/xylar/mambaforge/envs/test/lib/python3.10/site-packages/azure/kusto/data/client_base.py", line 20, in <module>
    from .kusto_trusted_endpoints import well_known_kusto_endpoints
  File "/home/xylar/mambaforge/envs/test/lib/python3.10/site-packages/azure/kusto/data/kusto_trusted_endpoints.py", line 107, in <module>
    _well_known_kusto_endpoints_data = json.load(open(_filename))
FileNotFoundError: [Errno 2] No such file or directory: '/home/xylar/mambaforge/envs/test/lib/python3.10/site-packages/azure/kusto/data/wellKnownKustoEndpoints.json'
```

#### Problem description

It seems the file `azure/kusto/datawellKnownKustoEndpoints.json` is not being packaged in the source as expected.

I believe the problem is that:
https://github.com/Azure/azure-kusto-python/blob/master/azure-kusto-data/setup.py#L46
should actually be:
``` python
    package_data={"azure/kusto/data/": ["wellKnownKustoEndpoints.json"]},
```
See the example in  https://setuptools.pypa.io/en/latest/userguide/datafiles.html#package-data.

This is causing the conda-forge build of 4.0.0 to fail:
https://github.com/conda-forge/azure-kusto-data-feedstock/pull/17

#### Output of ``pip freeze`` 

<details>

$ pip freeze
azure-core==1.26.1
azure-identity==1.12.0
azure-kusto-data @ file:///home/xylar/Desktop/azure/azure-kusto-data-4.0.0
certifi==2022.9.24
cffi==1.15.1
charset-normalizer==2.1.1
cryptography==38.0.3
idna==3.4
ijson==3.1.4
msal==1.20.0
msal-extensions==1.0.0
portalocker==2.6.0
pycparser==2.21
PyJWT==2.6.0
python-dateutil==2.8.2
requests==2.28.1
six==1.16.0
typing_extensions==4.4.0
urllib3==1.26.12

</details>

**Comment by xylar** [comment]
@AsafMah, thank you for addressing this!

**Comment by AsafMah** [comment]
Fixed in 4.0.1, let us know if you encounter further issues.

**Comment by xylar** [comment]
Will do!
